src/angularComponent/ngcOmniboxController.js

```diff
@@ -145,6 +145,25 @@ export default class NgcOmniboxController {
     this.highlightedIndex = -1; // -1 means nothing is highlighted
   }
 
+  /**
+   * Highlights a particular suggestion item.
+   *
+   * @param {Object} item
+   */
+  highlightItem(item) {
+    const match = this._suggestionsUiList.find((listItem) => listItem.data === item);
+
+    if (match) {
+      this.highlightedIndex = match.index;
```

Comment:
Actually no, nevermind. This function gets called from a mouse event on a suggestion item, and that only has access to its specific data not the "ui suggestion" that is in the flat array. I'll have to do a lookup to get the flat array's version unless you can think of a faster way.